intake_bluesky/jsonl.py

```diff
@@ -0,0 +1,224 @@
+import event_model
+from functools import partial
+import intake
+import intake.catalog
+import intake.catalog.local
+import intake.source.base
+import json
+from mongoquery import Query
+
+from .core import parse_handler_registry
+
+
+class BlueskyJSONLCatalog(intake.catalog.Catalog):
+    def __init__(self,  jsonl_filelist, *,
```

Comment:
Quick suggestion: all DataSources, even catalog ones, should have a name

Reply:
Thanks! Would you expect a generic `name` defined as class variabl,e [as in `CSVSource.name`](https://github.com/ContinuumIO/intake/blob/4557496793a32cc53397e5f2771992f9f999b605/intake/source/csv.py#L21), or an instance-specific one passed in to `__init__`?

Reply:
I meant a name for the class. It can be used in the `driver:` part of a data source spec, and appears as an `intake.open_*` function. An instance may have a different name.